src/test_workflow/integ_test/service_opensearch_dashboards.py

```diff
@@ -68,7 +68,7 @@ def url(self, path=""):
     def get_service_response(self):
         url = self.url("/api/status")
         logging.info(f"Pinging {url}")
-        return requests.get(url, verify=False, auth=("kibanaserver", "kibanaserver") if self.security_enabled else None)
+        return requests.get(url, verify=False, auth=("admin", "admin"))
```

Comment:
This is not correct you should not change the behavior of how to handle enabled security plugin, rather than only focus on the `api/status` I think.

Reply:
Also need @cliu123 to chime in on whether `api/status` requires `admin:admin`, or `kibanaserver:kibanaserver` is enough.

Reply:
Yes, `kibanaserver:kibanaserver` is good enough. Basically, any valid credential will go through authentication here. There is no authorization enforced here, so it doesn't matter what roles a user has. As long as the credential provided is valid, request goes through.

Reply:
Hey @cliu123 the `kibanaserver` user is commented out on default setup 
https://github.com/opensearch-project/security/blob/main/securityconfig/config.yml#L71
So correct me if i'm wrong `kibanaserver:kibanaserver` wont work I guess, unless we enable the right role for the index?

Reply:
@prudhvigodithi https://github.com/opensearch-project/opensearch-build/blob/main/config/opensearch_dashboards.yml#L178-L179

Reply:
Thanks @cliu123 for the clarification here.
@zelinh please restore.

Thanks.

Reply:
Thanks all for the inputs here. Reverted the changes here on integ test. 
@peterzhuamazon Please help review again.